security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/RelativeSecondsCodecs.scala

```diff
@@ -2,9 +2,10 @@ package pl.touk.nussknacker.ui.security.oauth2
 
 import io.circe._
 
+import java.time.Instant
 import scala.concurrent.duration.{Deadline, FiniteDuration, SECONDS}
 
-protected[oauth2] trait EpochSecondsCodecs {
+protected[oauth2] trait RelativeSecondsCodecs {
```

Comment:
I think that the main point is that it is not an epoch but duration measured in seconds. So maybe it should be rather `DurationInSecondsCodecs". Also why we still have deadline decoder here? Where it is used?

Reply:
yes, let's remove Deadline decoder altogether